### PR TITLE
Add modules and remove Home pages.

### PR DESCRIPTION
**Pull request by ADjenkov**
Add modules for `singup`/`login` features.
Remove Home pages
Add missing .gitignore file to singup folder
Add Logout handler in Login page

### CODE REVIEW

File: signup/app/signup/signup.component.ts [hunk 1]
```diff
@@ -13,20 +12,17 @@ export class SignupComponent implements OnInit {
     email: string;
     password: string;
 
-    constructor(private routerExtensions: RouterExtensions) {
+    constructor() {
```
Comment (manoldonev) [inline signup/app/signup/signup.component.ts:15]: We can remove the empty constructor.

File: login/app/login/login.component.ts [hunk 2]
```diff
@@ -12,16 +11,17 @@ export class LoginComponent implements OnInit {
     email: string;
     password: string;
 
-    constructor(private routerExtensions: RouterExtensions) {
+    constructor() {
```
Comment (manoldonev) [inline login/app/login/login.component.ts:14]: We can remove the empty constructor.

**Review by manoldonev — APPROVED** [review]

File: signup/app/signup/signup.component.ts (hunk 1 above)
Comment (ADjenkov) [reply signup/app/signup/signup.component.ts:15]: I thought that we want empty methods and constructors? 
To show the people where to put their stuff.

Comment (manoldonev) [reply signup/app/signup/signup.component.ts:15]: As discussed we keep them and will add comment inside.

**Pull request merged**